development-guide.md

```diff
@@ -34,25 +34,26 @@ Partners should not add the proto files to their repos. Proto files should be pu
 ```
 
 - Try to make log messages as _precise_ as possible, which can make it easier to debug issues. 
-- Provide context in log messages. Contextual information can make it much easier to debug issues
-- Write a separate error message for each exception
+- Provide context in log messages. Contextual information can make it much easier to debug issues.
+- Write a separate error message for each exception.
 - Log _after_ an action. When you log after an action, additional context can be provided.
 - Include details about "what went wrong" in your error message
-- Manage log volume. Ask yourself if a consumer of the log message will find it useful to solve a problem and whether that need justifies the cost of storing and managing that log. Sources of excessive logging include: 
+- Manage the volume of the log. Ask yourself if a consumer of the log message will find it useful to solve a problem and whether that need justifies the cost of storing and managing that log. Sources of excessive logging include: 
```

Comment:
I prefer the original here

Reply:
@chloe your take?

Reply:
@ediril 'log volume' as I read this may assume there's a dedicated data volume dedicated to logs. I may be wrong, though, and there's no confusion.
Just consider 
![Screenshot 2024-08-29 at 05 31 22](https://github.com/user-attachments/assets/3af678cb-022c-4e95-8305-1513551af22a)

Reply:
🤷‍♂️ I still prefer the original, but it is upto you